sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -233,6 +229,21 @@ public virtual TokenCredential Credential
                             clientSecret,
                             new ClientSecretCredentialOptions { AuthorityHost = new Uri(AuthorityHostUrl) });
                     }
+                    else if (!string.IsNullOrWhiteSpace(systemAccessToken))
+                    {
+                        // These variables should only be defined in the Live Test Pipelines so there is no need to record these variables
+                        _credential = new AzurePipelinesCredential(
+                                GetVariable("AZURESUBSCRIPTION_TENANT_ID"),
+                                GetVariable("AZURESUBSCRIPTION_CLIENT_ID"),
+                                GetVariable("AZURESUBSCRIPTION_SERVICE_CONNECTION_ID"),
```

Comment:
I believe these env vars only exist in the [sub-config-identity-test-resources](https://github.com/Azure/azure-sdk-for-net/blob/9d19a8531313909a932ad35cee9ea65e1ba67e1f/sdk/identity/tests.yml#L32) pipeline secret. They would need to be moved to the main [sub-config-azure-cloud-test-resources](https://github.com/Azure/azure-sdk-for-net/blob/9d19a8531313909a932ad35cee9ea65e1ba67e1f/sdk/identity/tests.yml#L30)  for this to work outside of the identity pipeline.

In addition, we'd need to add the [SYSTEM_ACCESSTOKEN](https://github.com/Azure/azure-sdk-for-net/blob/9d19a8531313909a932ad35cee9ea65e1ba67e1f/sdk/identity/tests.yml#L34-L35) globally

Reply:
These environment variables have been defined on the pipeline task running the tests.

Reply:
You mean pipelines that intend to use this will define them, or they are already defined? I don't see them anywhere in our tasks.

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/jobs/live.tests.yml#L182

@danieljurek recently started setting these in the step we are running our live tests.